src/System.Data.SqlClient/dir.props

```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.1.1.0</AssemblyVersion>
+    <AssemblyVersion>4.1.1.1</AssemblyVersion>
```

Comment:
I just pushed in a commit to fix the package errors. 
Added `<PackageVersion>4.3.1</PackageVersion>` to dirs.props